xml/System/String.xml

```diff
@@ -1735,10 +1737,13 @@ You can download the [Sorting Weight Tables](https://www.microsoft.com/download/
    
   
 ## Examples  
+
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```

Comment:
You don't need this. You've already add it to the Remarks section.

```suggestion
```

Reply:
Done